test/integration/targets/inventory_vmware_vm_inventory/runme.sh

```diff
@@ -49,22 +49,33 @@ curl "http://${VCENTER_HOST}:5000/govc_find"
 # Get inventory
 ansible-inventory -i ${VMWARE_CONFIG} --list
 
+echo "Check if cache is working for inventory plugin"
+if [ ! -n "$(find "${inventory_cache}" -maxdepth 1 -name 'vmware_vm_inventory_*' -print -quit)" ]; then
```

Comment:
In this old version of inventory_vmware_vm_inventory/runme.sh, ${inventory_cache } isn't defined.  It was defined in 8ce09a0057189733c814b01d0ecfa7ec405014a0  and it looks like you can just use ```$(pwd)/inventory_cache``` if you don't want to backport that.

Reply:
Thanks for tracking this down. I pushed a fix.